test/e2e/main_test.go

```diff
@@ -78,19 +80,46 @@ func TestMain(m *testing.M) {
 	os.Exit(m.Run())
 }
 
+type InstallTestCase struct {
+	Name      string
+	ExpectErr bool
+	TestFunc  func(t *testing.T, testReportingFramework *reportingframework.ReportingFramework)
+}
+
 func TestInstallMeteringAndReportingProducesData(t *testing.T) {
```

Comment:
We'll have to pick a better name for this now that it's doing a ton of different things in subtests

Reply:
`TestMeteringManualInstall` maybe?